lib/sda/node.go

```diff
@@ -347,9 +359,15 @@ func (n *Node) DispatchMsg(sdaMsg *SDAData) error {
 		dbg.Lvl4("Dispatching to handler", n.Entity().Addresses)
 		err = n.DispatchHandler(msgs)
 	default:
-		return errors.New("This message-type is not handled by this protocol")
+		err = errors.New("This message-type is not handled by this protocol")
+	}
+	// XXX Should at some point define a central way / point to handle errors
+	// ... ?
+	// Before the error was just going up to the sda.Host and nothing else was
+	// taken into account.
+	if err != nil {
+		dbg.Error(err)
```

Comment:
At least print it out (silently omitting it, can create boring, hence seemingly endless debugging sessions). Pass it up if you think there are reasons why the caller should handle it.